### ISSUE TITLE
Highlighted counter is broken on dark mode

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/nextcloud-vue#3249]
Items are:
* Outlined
* Highlighted
* Normal

Bright | Dark
---|---
![Bildschirmfoto vom 2022-09-14 09-48-33](https://user-images.githubusercontent.com/213943/190093736-afa320bb-1dff-488f-9bc2-69d2bf0e2e6c.png) | ![Bildschirmfoto vom 2022-09-14 09-48-23](https://user-images.githubusercontent.com/213943/190093730-40a4221e-ad12-4577-aaf2-7659d3363af8.png)

The difference between backgrounds of highlighted and normal is non-existant and especially the missing contrast of the font on normal is an issue

**Comment by nickvergessen** [comment]
Related server PR that removed at least the `-lighter` var which is used.
https://github.com/nextcloud/server/pull/33946

But even when restoring the var the look is not good

**Comment by nickvergessen** [comment]
Seems to be a problem with the math of the variables in server.
@skjnldsv is working on a patch
